be/src/vec/common/cow.h

```diff
@@ -350,8 +350,8 @@ class COW {
         const T& operator*() const { return *value; }
         T& operator*() { return value->assume_mutable_ref(); }
 
-        operator const immutable_ptr<T>&() const { return value; }
-        operator immutable_ptr<T>&() { return value; }
```

Comment:
Maybe you should update your formatter. https://github.com/apache/incubator-doris/pull/9602

Reply:
ok